src/psp_lda.py

```diff
@@ -0,0 +1,112 @@
+import sys
+import json
+from collections import Counter
+
+# Make sure to Install NLTK and Gensim, download stopwords
+import nltk
+import gensim
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+
+# from nltk.stem.snowball import SnowballStemmer
+from gensim.utils import simple_preprocess
+
+# TO DO
+# 1) Fiddle around with params (including num of topics)
+# 2) Look into Split Test Method
+# 2a) Once we have 6000+ docs, use 5000 to train, the rest to test ??
+# 3) Look into issue of not all sites being in English
+
+
+def main():
+    with open(sys.argv[1], "r") as input_file:
```

Comment:
It would also be great to rework this so we use the entire dataset on the db, but this can come later if you want.